src/storage/store/engine_tests.rs

```diff
@@ -1170,33 +1179,7 @@ mod tests {
         assert!(proof.is_some());
         assert_eq!(proof.unwrap().fid, FID_FOR_TEST);
 
-        let fname_transfer = username_factory::create_transfer(
-            0,
-            fname,
-            Some(FARCASTER_EPOCH + 1),
-            Some(FID_FOR_TEST),
-        );
-
-        let state_change = engine.propose_state_change(
-            1,
-            vec![MempoolMessage::ValidatorMessage(ValidatorMessage {
-                on_chain_event: None,
-                fname_transfer: Some(fname_transfer.clone()),
-            })],
-        );
-        test_helper::validate_and_commit_state_change(&mut engine, &state_change);
-
-        let transfer_event = &event_rx.try_recv().unwrap();
-        assert_eq!(
-            transfer_event.r#type,
-            proto::HubEventType::MergeUsernameProof as i32
-        );
-
-        // don't insert an fname for fid 0 into the trie
-        assert!(!test_helper::key_exists_in_trie(
-            &mut engine,
-            &TrieKey::for_fname(0, fname)
-        ));
+        // todo: alternate signer for fname server testing so we can verify transferring to 0 nukes the fname
```

Comment:
This is hard to do because of the signature validation? See note below. I think `username_factory::create_transfer` should use custom hardcoded signer for tests, and the engine should pull in the public key to use for validations from config.